src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,92 @@
+// ----- BASE ELEMENTS ---------------
+
+.navbar {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
+  background-color: #000000;
+}
+
+.img-left {
+  float: left;
+  width: 159px;
+  height: 56px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.right-menu {
+  float: right;
+}
+
+.navbar a img {
+  width: 119px;
+  height: 24px;
+}
+
+.navbar a {
```

Comment:
Please change to `.eyebrow a {`.  This class name may change, so we'll update if needed.

Reply:
Changes have been done, Please review